recipe/bld.bat

```diff
@@ -1,12 +1,23 @@
+@echo on
+
 mkdir build && cd build
-cmake ^
-	-G "%CMAKE_GENERATOR%"                   ^
+cmake -LAH -GNinja                           ^
```

Comment:
is the `-LAH` for development? could be removed if so

Reply:
Removed